phpBB/styles/prosilver/theme/content.css

```diff
@@ -491,18 +491,17 @@ blockquote cite > div {
 
 /* Code block */
 .codebox {
-	padding: 3px;
 	border: 1px solid transparent;
 	font-size: 1em;
 	margin: 1em 0 1.2em 0;
-	overflow-x: scroll;
 	word-wrap: normal;
 }
 
 .codebox p {
 	text-transform: uppercase;
 	border-bottom: 1px solid transparent;
-	margin-bottom: 3px;
+	margin-bottom: 0;
+	padding: 3px 3px 0 3px;
```

Comment:
```
<div class="codebox">
<p>
Code: <a href="#" onclick="selectCode(this); return false;">Select all</a>
</p>
<pre><code>...</code></pre>
</div>
```

The codebox initially had 3px padding for all sides, but it was unwanted since it created margins for the scrollbars and so I removed it. Thus, I had to transfer the padding to the child elements which are `.codebox p` and `.codebox code` (as there's no selector for `<pre>`).

Which is why I didn't give it any bottom padding, but I'll proceed to change it now.